### PR TITLE
Updates to the Summit and Projects pages

### PR DESCRIPTION
**Pull request by amirmc**
- Added a link to the LA Summit review post and mentioned the Copenhagen summit.
- Update the blurb in the projects page to read better and follow the `moby/moby` repo a little more closely

### CODE REVIEW

**Comment by amirmc** [comment]
I'll wait a little while in case people have additional comments and then I'll merge this.

File: docs/summit.md [hunk 1]
```diff
@@ -7,13 +7,16 @@ layout: default
...
+Event registration is free as a DockerCon Add On, and you can [add it to your registration now](https://europe-2017.dockercon.com/moby-summit/) now.
+
+We look forward to seeing you in Copenhagen, Denmark!
 
-If there are topics you want to see covered in this summit, please send a pull request to modify [the agenda](/events/summit-agenda-sept-2017/).
 
 ### Previous Moby Summits recaps
 
+* [Moby Summit, September 2017](https://blog.mobyproject.org/moby-summit-los-angeles-recap-a41e6acf81f8)
 * [Moby Summit, June 2017](http://mobyproject.org/blog/2017/06/26/moby-summit-recap/)
```
Comment (tiborvass) [inline docs/summit.md]: @amirmc do you mind changing this to `https://blog.mobyproject.org/moby-summit-june-2017-recap-b7f9c44334a6` ?

File: docs/summit.md [hunk 2]
```diff
@@ -7,13 +7,16 @@ layout: default
...
 * [Moby Summit, May 2017](http://mobyproject.org/blog/2017/05/11/moby-summit-report/)
```
Comment (tiborvass) [inline docs/summit.md]: And this one to `https://blog.mobyproject.org/moby-summit-at-dockercon-2017-05-11-9d11f4e7c5a1`

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**